api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -83,16 +77,10 @@ private String convertClusterName(String workspaceId) {
     workspaceService.enforceWorkspaceAccessLevel(workspaceNamespace,
         workspaceId, WorkspaceAccessLevel.WRITER);
 
-
-    Cluster createdCluster;
-
     String clusterName = this.convertClusterName(workspaceId);
-    try {
-      // TODO: Replace with real workspaceNamespace/billing-project
```

Comment:
@blrubenstein I'm removing a several of these TODOs which look stale to me, because AFAICT we're already passing a real value here. Please double-check and let me know if I'm missing something.

Reply:
Yup, we did, in fact, fix that!